src/test_doubly_linked_list.py

```diff
@@ -0,0 +1,166 @@
+# -*- coding utf-8 -*-
+import pytest
+
+
+def test_none():
+    from doubly_linked_list import DoubleLinkedList
```

Comment:
Your tests are in dire need of fixtures. Instead of having to import, initialize and set up a DoubleLinkedList instance in every test function, you can establish a few fixtures which you can then bring into your test functions as a parameter. This would cull many, many lines of code.